backend/app/api/v1/chat.py

```
@@ -0,0 +1,278 @@
+
+        # Map task types to agent roles
+        task_agent_mapping = {
+            "contract_fill": "contract_generator",
+            "document_extract": "data_extraction",
+            "signature_send": "signature_tracker",
+            "file_search": "help_assistance",
+            "compliance_check": "compliance_checker",
+            "summary_generate": "summary_agent"
+        }
+
+        agent_role = task_agent_mapping.get(request.task_type, "help_assistance")
```

Ruff (0.12.2)

Local variable `agent_role` is assigned to but never used

Remove assignment to unused variable `agent_role`

(F841)